lib/output.js

```diff
@@ -285,7 +285,15 @@ const tile = function tile (tile) {
       }
     }
   }
-  return this;
+  // Format
+  if (is.defined(this.options.formatOut) && is.string(this.options.formatOut)) {
```

Comment:
`this.options.formatOut` should always have a string value and therefore will be defined.